### DISCUSSION TITLE
Cask eloston-chromium latest update

### DISCUSSION THREAD

**Discussion by backstab5983** [Homebrew/discussions#4754] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: d4444b563e24ac7c05a93121c464c02dfa04d44f
Last commit: 4 days ago
Core tap JSON: 31 Aug 11:37 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.5.1-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

This cask doesn't support auto-updatest or "latest". I see that the latest available on brew is `115.0.5790.170` while in the official GitHub the [latest for MacOS](https://github.com/ungoogled-software/ungoogled-chromium/releases/tag/116.0.5845.110-1) is `116.0.5845.110`.

Is there any way to make the update happen? Or one just has to wait that the version gets updated in Homebrew?

### Relevant casks

eloston-chromium

**Comment by cho-m** [comment]
For `eloston-chromium`, https://github.com/ungoogled-software/ungoogled-chromium-macos needs to first build a package and then Homebrew needs to update to that version before you can get update.

So, as long as upstream repo hasn't updated to newer copy available in https://github.com/ungoogled-software/ungoogled-chromium, we can't do anything. They are still on `115.0.5790.170` (see: https://github.com/ungoogled-software/ungoogled-chromium-macos/releases/latest)

We don't build any Chromium app (base `chromium`, `ungoogled-chromium`, etc) from source.
